Rules/Sandbox/Scripts/Sandbox_Rules.as

```diff
@@ -389,41 +383,4 @@ void Reset(CRules@ this)
 	this.set("core", @core);
 	this.set("start_gametime", getGameTime() + core.warmUpTime);
 	this.set_u32("game_end_time", getGameTime() + core.gameDuration); //for TimeToEnd.as
-
-	kegCount = 0;
-	mineCount = 0;
-}
-
-void onBlobCreated(CRules@ this, CBlob@ blob)
```

Comment:
Because there are a hundred items and if you spawn one of them often enough it will lag the server to an unplayable state, which is what this PR aims to solve. A two hundred variables (one for counting, one for setting a limit) is bad.

This PR also makes it so a blob isn't created to begin with instead of creating and then instantly destroying it. That would cause player to lose coins and having bought nothing. When limiting stuff, limiting per team is necessary so as to avoid situations where one team has all the things and the other team can't make more.

(Yes, this is Sandbox but still.)